core/components/atoms/metricInput/MetricInput.tsx

```diff
@@ -257,7 +257,14 @@ export const MetricInput = React.forwardRef<HTMLInputElement, MetricInputProps>(
   const actionButtonSize = size === 'large' ? 'regular' : 'tiny';
 
   return (
-    <div data-test="DesignSystem-MetricInputWrapper" className={classes} onKeyDown={onKeyDown} role="presentation">
+    <div
+      data-test="DesignSystem-MetricInputWrapper"
+      className={classes}
+      onKeyDown={onKeyDown}
+      role="button"
```

Comment:
Please use the appropriate role based on component usage. Role here should be `textbox`.

Reply:
done